src/Components/Header/Header.jsx

```diff
@@ -1,45 +1,70 @@
 import { Link, useLocation } from "react-router-dom";
 import { useAuthContext } from "../../Context/Auth.context";
+import { useReducerContext } from "../../Context/Reducer.context";
+import { useThemeContext } from "../../Context/Theme.context";
 import { LoginBox } from "../../Pages/AuthPage/ProfileComponents";
 import "./Header.css";
 
 export const Header = () => {
+  const { theme, toggleLightDarkTheme } = useThemeContext();
   const user = localStorage.getItem("StormKeepUser");
   const encodedToken = localStorage.getItem("StormKeepToken");
```

Comment:
could make a custom hook for  using localstorage.

Reply:
Will think on this